javalib/src/main/scala/java/util/HashMap.scala

```diff
@@ -0,0 +1,187 @@
+package java.util
+
+import java.{util => ju}
```

Comment:
This file is in `java.util`. There's no need to import it. Just use unqualified names to refer to other classes of `java.util`.